app/views/shared/_place_card.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="card">
+  <% if local_assigns[:image_url].present? %>
+    <img class="card-img-top img-fluid" src="<%= image_url %>" alt="<%= name %>" style="height: 600px; object-fit: cover;">
```

Comment:
Outside of React, we generally make the changes to the CSS file instead of "inline styles" for performance and maintainability.